src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PEParameterSymbol.cs

```diff
@@ -645,6 +647,38 @@ internal override bool IsCallerMemberName
             }
         }
 
+        internal override bool NotNullWhenFalse
+        {
+            get
+            {
+                const WellKnownAttributeFlags flag = WellKnownAttributeFlags.NotNullWhenFalse;
+
+                bool value;
+                if (!_packedFlags.TryGetWellKnownAttribute(flag, out value))
+                {
+                    value = _packedFlags.SetWellKnownAttribute(flag,
+                        _moduleSymbol.Module.HasAttribute(_handle, AttributeDescription.NotNullWhenFalseAttribute));
+                }
+                return value || HasExtraAttribute(AttributeDescription.NotNullWhenFalseAttribute);
```

Comment:
Should `HasExtraAttribute(...)` be included in the `_packedFlags` value, to avoid calling `HasExtraAttribute` each time?